packages/@lwc/template-compiler/src/codegen/formatters/module.ts

```diff
@@ -15,41 +12,39 @@ import {
     LWC_MODULE_NAME,
 } from '../../shared/constants';
 
+import CodeGen from '../codegen';
 import { identifierFromComponentName, generateTemplateMetadata } from '../helpers';
+import State from '../../state';
 
-function moduleNameToImport(name: string): t.ImportDeclaration {
-    const localIdentifier = identifierFromComponentName(name);
+function generateComponentImports(codeGen: CodeGen): t.ImportDeclaration[] {
+    return Array.from(codeGen.referencedComponents).map((name) => {
+        const localIdentifier = identifierFromComponentName(name);
 
-    return t.importDeclaration(
-        [t.importDefaultSpecifier(localIdentifier)],
-        t.literal(kebabcaseToCamelcase(name))
-    );
+        return t.importDeclaration(
+            [t.importDefaultSpecifier(localIdentifier)],
+            t.literal(kebabcaseToCamelcase(name))
+        );
+    });
 }
 
-function generateSecureImports(additionalImports: string[]): t.ImportDeclaration {
-    const imports = additionalImports.map((additionalImport) => {
-        return t.importSpecifier(t.identifier(additionalImport), t.identifier(additionalImport));
+function generateLwcApisImport(codeGen: CodeGen): t.ImportDeclaration {
+    const imports = Array.from(codeGen.usedLwcApis).map((name) => {
+        return t.importSpecifier(t.identifier(name), t.identifier(name));
     });
```

Comment:
```suggestion
    const imports = Array.from(codeGen.usedLwcApis).sort().map((name) => {
        return t.importSpecifier(t.identifier(name), t.identifier(name));
    });
```

Sorting this array would result in more consistent and cleaner-looking generated code, and plus it would avoid modifying the existing snapshots (which have e.g. `import { registerTemplate, sanitizeAttribute } from "lwc";`)